index.html

```diff
@@ -16,16 +16,7 @@
       <div id="container-section">
         <div class="left-container">
           <div class="screen" id="screen-desktop">Screen</div>
-          <div class="buttons-container">
-            <button value="65">A</button>
-            <button value="83">S</button>
-            <button value="68">D</button>
-            <button value="70">F</button>
-            <button value="71">G</button>
-            <button value="72">H</button>
-            <button value="74">J</button>
-            <button value="75">K</button>
-            <button value="76">L</button>
+          <div class="buttons-container" id="buttonsContainer">
```

Comment:
@Aashishkushwaha i have dynamically created this in js file

Reply:
Thanks for your help.